docs/pds4_create_collection_product.rst

```diff
@@ -0,0 +1,52 @@
+``pds4_create_collection_product`` Program
+=================================
+
+Introduction
+------------
+The RMS Node's PDS4 Collection Product Creation Tool (``pds4_create_collection_product``)
+is designed to create bespoke collection product files from
+`Planetary Data System (PDS) <https://pds.nasa.gov>`_ `PDS4-format
+<https://pds.nasa.gov/datastandards/documents/>`_label files. This tool automates the
+creation of collection products by recursively searching for all label files within a
+specified directory and extracting their identifying information. This information is then
+compared against user-provided details (the bundle and collection names) to determine
+which label files are primary members and which are secondary members. The results are
+compiled into a CSV file and placed in the user's specified location, or in the
+collection's location if no specific destination is provided. This allows for a quick and
+efficient way to check the contents of collections.
+
```

Comment:
Say something about us not providing label files for the collection product, and the provider needing to work with their assigned Node to develop an appropriate label.